File: app/src/main/java/com/onepay/miuralibrary/data/TransactionData.java
package com.onepay.miuralibrary.data;

public class TransactionData {
    private String deviceCode;
    private String entryMode;
    private String  encryptedCardData;
    private float amount;
    private boolean returnStatus;
    private String returnReason;
    private String cardHolderName;
    private String cardNumber;
    private String accountFirstFour;
    private String accountLastFour;
    private String expiryDate;
    private String deviceId;
    private String kSN;
    private String maskedTrack2Data;

    public String getDeviceCode() {
        return deviceCode;
    }

    public void setDeviceCode(String deviceCode) {
        this.deviceCode = deviceCode;
    }

    public String entryMode() {
        return entryMode;
    }

    public void setEntryMode(String entryMode) {
        this.entryMode = entryMode;
    }

    public String cardData() {
        return encryptedCardData;
    }

    public void setCardData(String cardData) {
        this.encryptedCardData = cardData;
    }

    public float amount() {
        return amount;
    }

    public void setAmount(float amount) {
        this.amount = amount;
    }

    public boolean returnStatus() {
        return returnStatus;
    }

    public void setReturnStatus(boolean returnStatus) {
        this.returnStatus = returnStatus;
    }

    public String cardHolderName() {
        return cardHolderName;
    }

    public void setCardHolderName(String cardHolderName) {
        this.cardHolderName = cardHolderName;
    }

    public String cardNumber() {
        return cardNumber;
    }

    public void setCardNumber(String cardNumber) {
        this.cardNumber = cardNumber;
    }

    public String accountFirstFour() {
        return accountFirstFour;
    }

    public void setAccountFirstFour(String accountFirstFour) {
        this.accountFirstFour = accountFirstFour;
    }

    public String accountLastFour() {
        return accountLastFour;
    }

    public void setAccountLastFour(String accountLastFour) {
        this.accountLastFour = accountLastFour;
    }

    public String expiryDate() {
        return expiryDate;
    }

    public void setExpiryDate(String expiryDate) {
        this.expiryDate = expiryDate;
    }

    public String deviceId() {
        return deviceId;
    }

    public void setDeviceId(String deviceId) {
        this.deviceId = deviceId;
    }

    public String KSN() {
        return kSN;
    }

    public void setKSN(String kSN) {
        this.kSN = kSN;
    }

    public String getReturnReason() {
        return returnReason;
    }

    public void setReturnReason(String returnReason) {
        this.returnReason = returnReason;
    }

    public String getMaskedTrack2Data() {
        return maskedTrack2Data;
    }

    public void setMaskedTrack2Data(String maskedTrack2Data) {
        this.maskedTrack2Data = maskedTrack2Data;
    }

}

File: OnePayMiuraXamarin/OnepayMiura/Data/TransactionData.cs
using System;

namespace Onepay.Miura.Data
{
    public class TransactionData
    {
        private String deviceCode;
        private String entryMode;
        private String encryptedCardData;
        private double amount;
        private int returnStatus;
        private String returnReason;
        private String cardHolderName;
        private String cardNumber;
        private String accountFirstFour;
        private String accountLastFour;
        private String expiryDate;
        private String deviceId;
        private String kSN;
        private String maskedTrack2Data;

        public string DeviceCode { get => deviceCode; set => deviceCode = value; }
        public string EntryMode { get => entryMode; set => entryMode = value; }
        public string EncryptedCardData { get => encryptedCardData; set => encryptedCardData = value; }
        public double Amount { get => amount; set => amount = value; }
        public int ReturnStatus { get => returnStatus; set => returnStatus = value; }
        public string ReturnReason { get => returnReason; set => returnReason = value; }
        public string CardHolderName { get => cardHolderName; set => cardHolderName = value; }
        public string CardNumber { get => cardNumber; set => cardNumber = value; }
        public string AccountFirstFour { get => accountFirstFour; set => accountFirstFour = value; }
        public string AccountLastFour { get => accountLastFour; set => accountLastFour = value; }
        public string ExpiryDate { get => expiryDate; set => expiryDate = value; }
        public string DeviceId { get => deviceId; set => deviceId = value; }
        public string KSN { get => kSN; set => kSN = value; }
        public string MaskedTrack2Data { get => maskedTrack2Data; set => maskedTrack2Data = value; }
    }
}